docs/messaging-implementation.md

```diff
@@ -55,6 +56,7 @@ point uportal-app-framework to your desired feed.
 - **expireDate**: *(optional)* Accepts a simple ISO date, including time (as pictured). This is used to stop displaying a message at a certain day/time.
 - **featureImageUrl**: *(optional)* Used by popup announcements and announcements on the Features page.
 - **priority**: Set to "high" if you want the message to be displayed with higher visibility (i.e. As a priority notification or popup announcement, respectively).
+- **recurrence**:(optional) If true, even if a notification is dismissed, it will continue to reoccur in the user's home at the start of every session until the user is no longer a member of the targeted group. For example, if a user is a member of students-with-outstanding-parking-tickets, that user will be confronted with the notification at every login until they pay the fine.
```

Comment:
If this is a separate field, let's call it out as, like `dismissible`, "experimental", so that we remember why we don't have to semantically version evolutions of it for now.